modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -77,22 +77,23 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 +
 [NOTE]
 ====
-If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`.
+If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`. If you need to enable `zfcp.allow_lun_scan=1` because you are using a CSI driver, for example, you must configure your NPIV so that each node cannot access the boot partition of another node.
```

Comment:
```suggestion
If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`. If you must enable `zfcp.allow_lun_scan=1` because you use a CSI driver, for example, you must configure your NPIV so that each node cannot access the boot partition of another node.
```

> ! need to _verb_
Use "need to" only when "must" is too restrictive. For example, "you do not need to" is not the same as "you
must not".

> Use simple verbs and tenses.